### PR TITLE
Fixes #34600 - add web/cockpit console to host's kebab

### PR DESCRIPTION
**Pull request by amirfefer**
This adds the missing `web console` button from the legacy host page.

### CODE REVIEW

**Comment by adamruzicka** [comment]
Needs a rebase

**Comment by amirfefer** [comment]
@adamruzicka, thanks, rebased.

**Comment by amirfefer** [comment]
@adamruzicka, I think the spinner is a bit misleading here on every click, so I removed it - when the kebab menu opens, it is mounted and rendered, when the menu is hidden the menu is unmounted, however, if the API result keeps the same, it won't trigger any re-render.

**Comment by ezr-ondrej** [comment]
> if the API result keeps the same, it won't trigger any re-render.

Tho API request still happens, right? And that is significantly slower then re-render, so I'd not mind having re-render, but I do mind about too many API requests (if we can help it). Isn't there a way how to store the cockpit url into a host API response that is being already queried and stored?

**Comment by amirfefer** [comment]
> And that is significantly slower then re-render

The API still happens asynchronously to the browser rendering, so it won't cause any UI slowness, though it makes sense to move the `cockpit_url` to the host API 👍

**Comment by amirfefer** [comment]
I've added the `cockpit_url` attribute to the host API, instead of fetching it from a separate API call.
There is no extra API call when a user clicks on the kebab menu.

File: webpack/react_app/components/HostKebab/KebabItems.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { DropdownItem } from '@patternfly/react-core';
+import { CodeIcon } from '@patternfly/react-icons';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { HOST_DETAILS_KEY } from 'foremanReact/components/HostDetails/consts';
+
+const HostKebabItems = () => {
+  const { cockpit_url: consoleUrl } = useSelector(state =>
```
Comment (amirfefer) [inline webpack/react_app/components/HostKebab/KebabItems.js:10]: There is no trivial way to propagate the `cockpit_url` from core,  for an easy cherry-pick the attribute is taken directly from redux's store.

Comment (ezr-ondrej) [reply webpack/react_app/components/HostKebab/KebabItems.js:10]: In this case it's IMHO fine as it is not one of main attributes :)

**Review by ezr-ondrej — APPROVED** [review]

File: app/models/concerns/foreman_remote_execution/host_extensions.rb [hunk 2]
```diff
@@ -46,6 +46,10 @@ def search_by_job_invocation(key, operator, value)
       end
     end
 
+    def cockpit_url
+      SSHExecutionProvider.cockpit_url_for_host(self.name)
```
Comment (ezr-ondrej) [inline app/models/concerns/foreman_remote_execution/host_extensions.rb:50]: This was recently renamed, but we did not update all usages anyway and in order for this to be cherry-pickable, I'm happy we are using the old name here :+1:

**Review by ezr-ondrej — APPROVED** [review]
Looks good, works well what's not to like? :)

**Pull request merged**